src/parameter_declarations.ts

```diff
@@ -466,3 +469,9 @@ Parameter.declare({
 	description:
 		'(OPTIONAL) Provide a custom content type to all files within the upload to be used by the gateway to display the content'
 });
+
+Parameter.declare({
+	name: CustomTagsParameter,
+	aliases: ['--custom-tags'],
```

Comment:
Consider using `--custom-tags-file` or similar. We might end up implementing an extra `--custom-tags` of type array later on

Reply:
Changed this to `--metadata-file` and `--metadata-json` + `metadata-gql-tags`